cmd/release-controller/audit_backend_gcs.go

```diff
@@ -126,7 +125,7 @@ func (b *GCSAuditStore) PutSignature(ctx context.Context, dgst string, signature
 		objectPath = path.Join("signatures", "openshift", "release", fmt.Sprintf("%s=%s", parts[0], parts[1]), fmt.Sprintf("signature-%d", index))
 	}
 	glog.V(4).Infof("Writing signature to gs://%s/%s", b.bucketName, objectPath)
-	obj := b.bucket.Object(objectPath).WithConditions(storage.IfGenerationMatch(0))
+	obj := b.bucket.Object(objectPath).If(storage.Conditions{GenerationNotMatch:1})
```

Comment:
@smarterclayton please review this

Reply:
Generation can be more than one if someone has edited the signature, is there a `storage.Conditions` for `GTE`?